### PR TITLE
Add emoji cache

### PR DESCRIPTION
**Pull request by Ratismal**
Now without recreating the collection every time it's called.

👀

### CODE REVIEW

File: lib/Client.js [hunk 1]
```diff
@@ -138,7 +140,7 @@ class Client extends EventEmitter {
             game: null,
             status: "offline"
         };
-
+
```
Comment (abalabahaha) [inline lib/Client.js]: :eyes:

**Comment by abalabahaha** [comment]
Again, I still don't like the idea of a Client-wide emoji cache.

Also, you're missing the initial GUILD_CREATE/READY emojis

**Comment by Ratismal** [comment]
I'm not missing GUILD_CREATE and READY emojis, as it updates every time the `guildCreate()` function is called.

I *am* missing the GUILD_DELETE emojis, and will push those soon.

I will not give up on this. ❤

**Comment by abalabahaha** [comment]
Do it in this PR then, not a new one

**Pull request closed without merging**